types/set/nested/LinkDef.h

```diff
@@ -0,0 +1,12 @@
+#include <cstdint>
+#include <set>
+
+#ifdef __CLING__
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ nestedclasses;
```

Comment:
Looking at `LinkdefReader.cxx`, it says `we don't really process that one` - what does it do?